v1/sensor3700.js

```diff
@@ -15,15 +14,142 @@ module.exports = function (RED) {
     node.reactivePower = config.reactivePower;
     node.reactiveEnergy = config.reactiveEnergy;
     node.relay = config.relay;
-    node.observationInterval = Number(config.interval);
+    node.observe_time = config.interval;
     node.name = config.uuid;
     node.paths = [];
     node.device = new restAPI.Device(node.service.service, node.name);
     node.powerSourceVoltage = config.powerSourceVoltage;
     node.state = false;
     node.cache = {};
-    // node.message = {};
-    // node.message.payload = {};
+
+    function configure() {
+      if (node.powerSourceVoltage) {
```

Comment:
change insides of ```configure()``` function to new version of resource writing (copy-paste from new version)